view/frontend/layout/smile_store_locator_store_view.xml

```diff
@@ -58,6 +57,7 @@
                             </item>
                         </argument>
                     </arguments>
+                    <block class="Smile\StoreLocator\Block\View\SetStoreLink" name="smile_store_locator_retailer_view.setstorelink" as="setstorelink" template="view/setStoreLink.phtml"/>
```

Comment:
Can you add `Smile_StoreLocator::view/setStoreLink.phtml` instead of `view/setStoreLink.phtml` ?